recipes/gensim/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "gensim" %}
+{% set version = "2.3.0" %}
+{% set sha256 = "7d0dccc7d2c576e270037949874800b7cfbc86ef081ff981483f612cd18e223f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.11.3
+    - scipy >=0.18.1
+    - six >=1.5.0
+    - smart_open >=1.2.1
+    
+  run:
+    - python
+    - numpy >=1.11.3
+    - scipy >=0.18.1
+    - six >=1.5.0
+    - smart_open >=1.2.1
+    
+test:
+  requires:
+    - morfessor
+    - scikit-learn
+    
+  imports:
+    - gensim
+    - gensim.corpora
+    - gensim.models
+    - gensim.models.wrappers
+    - gensim.parsing
+    - gensim.scripts
+    - gensim.similarities
+    - gensim.sklearn_integration
+    - gensim.summarization
+    - gensim.test
+    - gensim.topic_coherence
+    
+  commands:
```

Comment:
There are a few reasons for removing the test runs-
1) Tensorflow doesn't support 32-bit Windows, so running Gensim tests for win-32 on Appveyor will fail. It is possible to skip win-32 builds, but it might defeat the purpose of full support.

2) The test run times out in Travis( which is used for building recipe for OSX) at Doc2Vec parallel training test due to no output for 10 mins.